app/templates/suppliers/companies_house_number.html

```diff
@@ -28,12 +28,13 @@
 
   <div class="grid-row">
     <div class="column-two-thirds">
-      <div class="hint">
-        <p>This is a unique 8-character number given to your organisation by Companies House.</p>
-        <p><a href="https://beta.companieshouse.gov.uk/">Visit Companies House to get your number</a></p>
-      </div>
       <form method="POST" action="{{ url_for('.companies_house_number') }}">
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+        {% set question_advice =
+            'This is a unique 8-character number given to your organisation by Companies House.
+            \n\n<a href="https://beta.companieshouse.gov.uk/">Visit Companies House to get your number</a>
```

Comment:
So, you're right.  Doing that does work.

@Wynndow had a look and he was saying he thinks the `\n\n` is clearer because you're passing in the formatting information explicitly rather than needing to ensure on left-aligned text with the right number of line breaks separating them.

I think that's a reasonable viewpoint and I'm happy enough to leave the `\n`s even though it obviously looks a bit shit. 

What do you think?